tools/ReleaseEngineering/ServicingPipeline.ps1

```diff
@@ -61,6 +61,17 @@ Function Reject() {
     $Host.ExitNestedPrompt()
 }
 
+If ([String]::IsNullOrEmpty($Version)) {

```

Comment:
You can just write `if (!$Version)`.
(Small `if` is the canonical variant nowadays btw.)

Reply:
Ah! Didn't know that. Is that supported on PS 5.1?

Reply:
It seems to work. Is PS 5.1 still relevant though? We already require pwsh 7 or newer for the build anyways (to compress JSON strings).